cloud/pkg/dynamiccontroller/application/application.go

```diff
@@ -98,7 +98,6 @@ type Application struct {
 	count     uint64 // count the number of current citations
 	countLock sync.Mutex
 	tim       time.Time // record the last closing time of application, only make sense when count == 0
```

Comment:
you can fix this typo from `tim` to `time`

Reply:
done